iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergedLine.java

```diff
@@ -95,6 +97,33 @@ private void mergeProperties(DanglingLine dl1, DanglingLine dl2) {
         });
     }
 
+    void computeAndSetP0() {
+        double p1 = dl1.getTerminal().getP();
+        double p2 = dl2.getTerminal().getP();
+        if (!Double.isNaN(p1) && !Double.isNaN(p2)) {
+            double p0 = (p1 + p2) / 2.0d;
```

Comment:
I think there is a mistake here: the sum should use absolute value: p1 and p2 have opposite sign and their values are close. So if I make the sum, I will have a value close to 0...
Example:
```
p1 = -605
p2 = 600
--> losses = 5
p1 + p2 / 2.0 = -2,5
```

Another way (simplest) way to compute the P0 and Q0 is to use the losses:
```
double losses = (p1 + p2) / 2.0
dl1.P0 = (P1 + 1/2 losses) * sign(P2)
dl2.P0 = (P2 + 1/2 losses) * sign(P1)
```